src/ActivitiesSDKAdapter.js

```diff
@@ -20,21 +21,54 @@ import logger from './logger';
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/ActivitiesAdapter.js#L6}
  */
 
+/**
+ * Extracts JSON card and files from server activity.
+ *
+ * @private
+ * @param {object} sdkActivity  SDK activity object
+ * @returns {Array} Array of attachments.
+ */
+function parseSDKAttachments(sdkActivity) {
+  const {attachments = []} = sdkActivity;
+
+  if (sdkActivity.attachments) {
+    attachments.concat(sdkActivity.attachments);
+  }
+
+  // If object.files, then merge files.items into attachments.
+  if (sdkActivity.card) {
+    attachments.push(sdkActivity.card);
+  }
+
+  try {
+    if (sdkActivity.object && sdkActivity.object.cards) {
+      const card = JSON.parse(sdkActivity.object.cards[0]);
```

Comment:
from our tests, the cards are encrypted in the sdk activity object. They need to be decrypted.

Reply:
Ok, how would i go about doing this?

Reply:
After investigation, I did not see cards or attachments being returned encrypted from the API.